File: python/prism_sdk/federated_continual_knowledge_representation_workflow_fabric.py
"""Worldgen P04 AFA-worldgen-P04-F16 product wrapper."""
from .worldgen_knowledge_workflow_support import KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt, manifest as _manifest, schedule as _run
FEATURE_ID="AFA-worldgen-P04-F16"
CONTRACT_VERSION="worldgen-federated-continual-knowledge-workflow/1.0"
def worldgen_federated_continual_knowledge_representation_workflow_fabric_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeWorkflowRequest1@1", scale="federated continual autonomous", autonomy_tier="A2")
def schedule_worldgen_federated_continual_knowledge_representation_workflow(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="federated continual autonomous", require_approval=True, require_federation=True)
__all__=["KnowledgeWorkflowRequest","KnowledgeWorkflowReceipt","worldgen_federated_continual_knowledge_representation_workflow_fabric_manifest","schedule_worldgen_federated_continual_knowledge_representation_workflow"]

File: crates/worldgen/src/federated_continual_knowledge_representation_workflow_fabric.rs
//! Worldgen P04-F16 federated continual knowledge-representation workflow fabric.
use super::knowledge_workflow_support::{self, KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F16"; pub const CONTRACT_VERSION:&str="worldgen-federated-continual-knowledge-workflow/1.0";
pub fn worldgen_federated_continual_knowledge_representation_workflow_fabric_manifest()->serde_json::Value{knowledge_workflow_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeWorkflowRequest1@1","federated continual autonomous","A2")}
pub fn schedule_worldgen_federated_continual_knowledge_representation_workflow(request:&KnowledgeWorkflowRequest)->Result<KnowledgeWorkflowReceipt,knowledge_workflow_support::KnowledgeWorkflowError>{knowledge_workflow_support::schedule(request,FEATURE_ID,CONTRACT_VERSION,"federated continual autonomous",true,true)}
pub use knowledge_workflow_support::{KnowledgeWorkflowError,KnowledgeWorkflowReceipt as WorldgenFederatedContinualKnowledgeWorkflowReceipt,KnowledgeWorkflowRequest as WorldgenFederatedContinualKnowledgeWorkflowRequest};
